test/config/jobconfig.json

```diff
@@ -1,52 +1,53 @@
 [
   {
     "jobType": "archive",
-    "create": [
-      {
-        "action": {
-          "actionType": "log"
-        }
-      }
-    ],
-    "update": {
-      "action": {
+    "create": {
+      "actions": {
         "actionType": "log"
       }
     },
-    "post": [
-    {
-      "action": {
-        "actionType": "email",
-        "mailer": {
-          "host": "smtp.myinstitution.ext",
-          "port": 465,
-          "secure": true,
-          "auth": {
-            "user": "scicatuser",
-            "pass": ""
-          }
-        },
-        "to": "{{ join job.recipients ' '}}",
-        "from": "scicat@example.com",
-        "subject": "Archive job {{job.pid}} has completed.",
-        "body": "Dear {{job.owner}},\n\nArchive job {{job.pid}} has completed. Results: {{job.status}}\n"
+    "update": {
+      "auth": "#all",
+      "actions": {
+        "actionType": "log"
       }
     },
-    {
-      "filter": {
-        "status": "SUCCESSFUL"
-      },
-      "action": {
-        "type": "url",
-        "url": "myscicat.myinstitution.ext/datablock/archived",
-        "verb": "post",
-        "headers": {
-          "AuthBearer": "{{secrets.jwt_token}}"
+    "post": {
+      "actions": [
+        {
+          "actionType": "email",
+          "mailer": {
+            "host": "smtp.myinstitution.ext",
+            "port": 465,
+            "secure": true,
+            "auth": {
+              "user": "scicatuser",
+              "pass": ""
+            }
+          },
+          "to": "{{ join job.recipients ' '}}",
+          "from": "scicat@example.com",
+          "subject": "Archive job {{job.pid}} has completed.",
+          "body": "Dear {{job.owner}},\n\nArchive job {{job.pid}} has completed. Results: {{job.status}}\n"
         },
-        "body": "{ structure to create archive files }"
-      }
+        {
+          "actionType": "filter",
```

Comment:
Not yet implemented, but I think this is how we will implement action-level filters.

Reply:
Nice idea!!!